DRAFT_homogeneous_impact_areas.Rmd

```diff
@@ -0,0 +1,316 @@
+tmap_arrange(map_trd,
+             map_trd_reclassed,
+             ncol=1)
+```
+
+I tweaked the thresholds for the bins so that most of the forest next to Trondheim (to the left in the map) was in the second lowest class.
+This looks pretty good to me.
+It depicts a gradient in human presence from high within the buildt zone, to *no to very low* in the forests and mountains to the west.
+Note that there is still considerable human activity also there in the form of outdoor recreation and even forestry.
+
+#### Aggregate
+
+The resolution in this map is more than we need, and the size of the data implies that the next step, the vectorization, would take too long.
+We therefore aggregate, ar reduce the resolution.
+
+```{r}
+dim <- st_dimensions(infra)
+paste("Resolution is", dim$x$delta, "by", dim$x$delta, "meters")
+```
+
+```{r, warning=F, message=F}
+infra_trd_reclassed_agg <- st_warp(infra_trd_reclassed, cellsize = c(1000, 1000), 
+                                   crs = st_crs(infra_trd_reclassed), 
+                                   use_gdal = TRUE,
+                                   method = "average")
+```
+
+```{r}
+dim <- st_dimensions(infra_trd_reclassed_agg)
+paste("Resolution is", dim$x$delta, "by", dim$x$delta, "meters")
+```
+
+Each cell is now the average of the aggregated cells, and hence the value is continuous again.
+Let's make it discrete.
+
+```{r}
+names(infra_trd_reclassed_agg) <- "infrastructureIndex"
+infra_trd_reclassed_agg <-  infra_trd_reclassed_agg %>%
+  mutate(infrastructureIndex = case_when(
+    infrastructureIndex < 1 ~ 0,
+    infrastructureIndex < 6 ~ 1,
+    infrastructureIndex < 9 ~ 2,
+    infrastructureIndex >= 9 ~ 3
+  ))
+```
+
+
+```{r, fig.cap="Infrastrcture indeks over Trondheim on a four-step discrete scale, aggregated to 1x1 km resolution using the mean function."}
+tm_shape(infra_trd_reclassed_agg)+
+  tm_raster(title="Infrastructure Index",
+            style="cat")+
+  tm_layout(legend.outside = T)+
+  tm_shape(hw_utm)+
+  tm_lines(col="white")
+```
+
+This resolution is more that good enough for our purpose here.
+
+### Aggregate the entire map to 1x1 km
+
+```{r, eval=F}
+# runtime about 30 sec
+infra_agg <- st_warp(infra, cellsize = c(1000, 1000), 
+                                   crs = st_crs(infra), 
+                                   use_gdal = TRUE,
+                                   method = "average")
+saveRDS(infra_agg, "P:/41201785_okologisk_tilstand_2022_2023/data/cache/infra_agg.rds")
+```
+
+```{r, echo=F}
+infra_agg <- readRDS("P:/41201785_okologisk_tilstand_2022_2023/data/cache/infra_agg.rds")
+```
+
+
+### Discretize the entire map
+
+```{r}
+names(infra_agg) <- "infrastructureIndex"
+
+infra_agg_discrete <- infra_agg %>%
+  mutate(infrastructureIndex = case_when(
+    infrastructureIndex < 1 ~ 0,
+    infrastructureIndex < 6 ~ 1,
+    infrastructureIndex < 9 ~ 2,
```

Comment:
With this categorisation, the total area in Norway with infrastructureIndex == 2 is less then for infrastructureIndex == 3. Try changing the threshold from 9 to maybe 10-12.

Reply:
Fixed (see [here](7d17771cab9a6a52ce027491643fc85f16bff209)).